src/mngt/ocf_mngt_cache.c

```diff
@@ -938,16 +935,38 @@ static void _ocf_mngt_attach_load_properties_end(void *priv, int error,
 	context->metadata.shutdown_status = properties->shutdown_status;
 	context->metadata.dirty_flushed = properties->dirty_flushed;
 
-	if (cache->device->init_mode == ocf_init_mode_load) {
-		context->metadata.line_size = properties->line_size;
-		cache->conf_meta->metadata_layout = properties->layout;
-		cache->conf_meta->cache_mode = properties->cache_mode;
+	ocf_pipeline_next(context->pipeline);
```

Comment:
actually looks like this unused in current PR revision. both OCF_PL_FINISH_RET and OCF_PL_NEXT_RET do "return"